rule.raml

```diff
@@ -0,0 +1,364 @@
+#%RAML 0.8
+title: Rule
+version: v0.1.1-20180620
+documentation:
+
+  - title: OCF Copyright
+    content: Copyright (c) 2018 Open Connectivity Foundation, Inc. All rights reserved.
+  - title: OCF License
+    content: !include LICENSE.md
+  - title: OCF Disclaimer
+    content: !include DISCLAIMER.md
+
+
+schemas:
+  - rule-rw: !include schemas/oic.rule-rw-schema.json
+  - baseline: !include schemas/oic.core-schema.json
+  - batch: !include schemas/oic.batch-retrieve-schema.json
+  - linklist: !include schemas/oic.oic-linklist-schema.json
+  - linkupdate: !include schemas/oic.if.linkupdate-schema.json
+  - create: !include schemas/oic.if.create-schema.json
+  - create-response: !include schemas/oic.if.create-response-schema.json
+  - delete-Response: !include schemas/oic.if.delete-response-schema.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+          enum: ["oic.if.rw", "oic.if.ll", "oic.if.b", "oic.if.baseline", "oic.if.create", "oic.if.delete", "oic.if.linkupdate"]
+
+
+/RuleURI?if=oic.if.baseline:
+    description: |
+      Baseline representation of a rule contains its common properties
+    displayName: rule-baseline
```

Comment:
This does not follow the requirement that baseline returns ALL Properties as defined in the Core Spec.  I think you need to add "links" and all of the Rule Properties (e.g. ruleenable, rule, ruleresult, etc.).